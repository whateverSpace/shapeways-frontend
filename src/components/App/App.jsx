import React from 'react';
import Sketch from '../Sketch/Sketch';

export default function App() {
  return <Sketch />;
}
