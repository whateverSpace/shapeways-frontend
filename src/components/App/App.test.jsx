import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
  