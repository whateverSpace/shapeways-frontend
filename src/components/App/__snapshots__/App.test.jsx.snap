// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<h1>
  Hello World
</h1>
`;
